sdcm/k8s_configs/ingress-controller/10_haproxy-kubernetes-ingress.deploy.yaml

```diff
@@ -22,6 +22,15 @@ spec:
     spec:
       terminationGracePeriodSeconds: 70
       serviceAccountName: haproxy-kubernetes-ingress
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                labelSelector:
+                  matchLabels:
+                    app.kubernetes.io/name: haproxy-ingress
+                topologyKey: kubernetes.io/hostname
+              weight: 1
```

Comment:
Why do you need it?

Reply:
I don't know, but that was part of the diff we had with the cloud team configuration...

Reply:
It is garbage in our case. We set our own affinity rules making it be scheduled onto the "default" node pool.
Remove it please.

Reply:
we'll remove

Reply:
removed